### ISSUE TITLE
Error with 66 character length pubkey? 

### ISSUE THREAD

**Issue by Jasko77** [JeanLucPons/Kangaroo#95]
Hi,

I have used this program earlier, and Got it working with some keys, but when i tried To extract pub keys for 120bit with albertos keysubtracter, the pubkey which it posts out In file, doesnt work and throws an error.

ParsePublicKeyHex: Error invalid public key specified (66 character length)

Or should the file need To remove all characters after pub key(were separeted with space) 

https://github.com/albertobsd/keysubtracter

Any tips for here, how these could be used?  🤔

**Comment by WanderingPhilosopher** [comment]
Yes, you will need to extract just the 66 character pubkeys from the outputfile OR use the -x flag:

-x Exclude comments; the + and/or - columns. (You need the comments if using Random mode or else you will not know what to add or subtract back to the found key to find the target key.)
